valgrind/run-valgrind-tests

```diff
@@ -20,31 +31,30 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get install -yq \
 export LC_ALL=en_US.UTF-8
 export LANG=en_US.UTF-8
 export LANGUAGE=en_US.UTF-8
-export PG_CONFIG=/usr/local/pgsql/bin/pg_config
+export PATH=$HOME/pgsql/bin:$PATH
 
 # download and install PostgreSQL
-git clone -b "REL_10_STABLE" --depth 1 git://git.postgresql.org/git/postgresql.git
+git clone -b "${POSTGRES_GITREF}" --depth 1 git://git.postgresql.org/git/postgresql.git
 cd postgresql/
-./configure --enable-cassert --enable-debug CFLAGS="-ggdb -Og -DUSE_VALGRIND"
+./configure \
+    --prefix=$HOME/pgsql \
+    --with-openssl \
+    --enable-cassert \
+    --enable-debug \
+    CFLAGS="-ggdb -Og -DUSE_VALGRIND"
 
 # we will use this to parallelize PostgreSQL compilation
 procs="$(nproc)"
 mjobs="$((procs + 1))"
-make -j "${mjobs}" -s
-sudo make install
-export PATH=/usr/local/pgsql/bin:$PATH
+make install -j "${mjobs}" -s
 
 # download and install Citus
 cd ..
-git clone https://github.com/citusdata/citus.git
+git clone -b "${CITUS_GITREF}" --depth 1 https://github.com/citusdata/citus.git
 cd citus/
 ./configure
 make clean
-make -j8 -s
-sudo make install
-
-# this is necessary to start tests
-sudo chown ubuntu /usr/local/pgsql/bin/ -R
+make install -j8 -s
```

Comment:
This is forgotten in the original script as well but we can use mjobs here too like above.

Though, I don't remember why we are adding 1 to the nproc.

Reply:
Good one, I will replace.

re the adding 1, this is mostly done to utilise cpu better. When you have exactly the number of jobs as cores they are often waiting on the disk as compiling is both IO and cpu intensive. By overbooking the jobs on CPU's they are more busy.